coq-doc.opam

```diff
@@ -16,7 +16,7 @@ homepage: "https://coq.inria.fr/"
 doc: "https://coq.github.io/doc/"
 bug-reports: "https://github.com/coq/coq/issues"
 depends: [
-  "dune" {build & >= "2.5.0"}
+  "dune" {>= "2.9"}
```

Comment:
Was `build` incorrect?

Reply:
Until Dune 2.3 it was Ok, later on, the `dune-package` meta-data files became versioned, so indeed, if dune is updated the metadata needs to be regenerated.

This is a pitfall, tho the cache doesn't cache dune itself so it helps.